tools/introspection/CMakeLists.txt

```diff
@@ -44,7 +44,11 @@ endif()
 # cmake doesn't find ncurses under QNX; let the compiler complain if the header is missing
 if(NOT QNX)
     SET(CURSES_USE_NCURSES TRUE)
-    find_package(Curses REQUIRED)
+    find_package(Curses QUIET)
+    if(NOT CURSES_FOUND)
+        message(STATUS "The build of the introspection client is skipped due to missing the curses library")
+        return()
```

Comment:
Can we remove `if(NOT QNX)` as well as the one for `Windows` on line 25. It seems superfluous now when we skip the build anyway when the library is not found.

Reply:
@elBoberido Done, could you please re-review? :pray: